CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+### Contributing
+
+- When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change.
+
+### We Develop with Github
+We use github to host code, to track issues and feature requests, as well as accept pull requests.
+
+### We Use [Github Flow](https://guides.github.com/introduction/flow/index.html), So All Code Changes Happen Through Pull Requests
+- Pull requests are the best way to propose changes to the codebase (we use Github Flow). We actively welcome your pull requests:
+
+1. Fork the repo and create your branch from master.
+2. If you've added code that should be tested, add tests.
+3. If you've changed APIs, update the documentation.
+4. Ensure the test suite passes.
+5. Make sure your code lints.
+6. Issue that pull request!
+
+### Code of Conduct
```

Comment:
Can be added as a separate Markdown document as [in the Atom repository for example](https://github.com/atom/atom/blob/master/CODE_OF_CONDUCT.md). If we wish to stick to a limited version, then we can add a Contributor Covenant Badge to the README.

Reply:
Right.